waterlog/src/Pages/Usage.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import BtnResolve from '../Components/BtnResolve';
+import BtnResolve from '../Components/BtnResolve'; 
```

Comment:
keep these consistent with the rest of your import statements: i.e. " " instead of ' '. Fix in other classes as well

Reply:
Sure I fixed it.